pallets/automation-time/src/tests.rs

```diff
@@ -327,9 +329,19 @@ fn schedule_transfer_with_dynamic_dispatch() {
 		let recipient = AccountId32::new(BOB);
 		assert_eq!(Balances::free_balance(recipient.clone()), 127);
 
+		let mut condition: BTreeMap<String, String> = BTreeMap::new();
+		condition.insert(String::from("type"), String::from("time"));
+		condition.insert(String::from("timestamp"), format!("{}", SCHEDULED_TIME));
 		assert_eq!(
 			my_events,
 			[
+				RuntimeEvent::AutomationTime(crate::Event::TaskTriggered {
+					who: account_id.clone(),
+					task_id: task_id.clone(),
+					condition,
+					encoded_call: call.encode(),
+					abort_errors: vec![],
+				}),
```

Comment:
The below two events are not necessary in our test. Remember this, for each operation we only need to test one event, which is usually the last event. For example in this test is `Event::Transfer`. This way our automation tests don’t need to be too coupled with the logic of other pallets, and testing `Event::Transfer` is sufficient to make sure the encoded_call happen.

```
RuntimeEvent::System(frame_system::Event::NewAccount {
					account: recipient.clone()
				}),
				RuntimeEvent::Balances(pallet_balances::pallet::Event::Endowed {
					account: recipient.clone(),
					free_balance: 127,
				}),
```

Reply:
Fixed